var/spack/repos/builtin/packages/cloverleaf3d/package.py

```diff
@@ -6,59 +6,78 @@
 
 class Cloverleaf3d(MakefilePackage):
     """Proxy Application. CloverLeaf3D is 3D version of the
-       CloverLeaf mini-app. CloverLeaf is a mini-app that solves
-       the compressible Euler equations on a Cartesian grid,
-       using an explicit, second-order accurate method.
+    CloverLeaf mini-app. CloverLeaf is a mini-app that solves
+    the compressible Euler equations on a Cartesian grid,
+    using an explicit, second-order accurate method.
     """
 
     homepage = "http://uk-mac.github.io/CloverLeaf3D/"
-    url      = "http://downloads.mantevo.org/releaseTarballs/miniapps/CloverLeaf3D/CloverLeaf3D-1.0.tar.gz"
+    url = "http://downloads.mantevo.org/releaseTarballs/miniapps/CloverLeaf3D/CloverLeaf3D-1.0.tar.gz"
 
-    tags = ['proxy-app']
+    tags = ["proxy-app"]
 
-    version('1.0', sha256='78d591728c61bdfd6175b3930df7652e09ed04fbcd01b3fc86fb2aa0f237a8ef')
+    maintainers = ["s1913388"]
 
-    variant('openacc', default=False, description='Enable OpenACC Support')
+    version(
+        "1.0", sha256="78d591728c61bdfd6175b3930df7652e09ed04fbcd01b3fc86fb2aa0f237a8ef"
+    )
```

Comment:
why?

Reply:
The reformatting is presumably done by black. We're currently working on integrating black into Spack's tests, but until then I generally ignore style changes as long as they pass the tests.